### PR TITLE
 RAB: Integrate staging tests for the .some method

### PR DESCRIPTION
**Pull request by ioannad**
of Array.prototype and TypedArray.prototype

This is part of PR https://github.com/tc39/test262/pull/3888 to make reviewing easier. Includes changes to use the helper ./harness/resizableArrayBufferUtils.js

### CODE REVIEW

File: test/built-ins/Array/prototype/some/resizable-buffer.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  //                    [4, 6] << fixedLengthWithOffset
+  //              [0, 2, 4, 6, ...] << lengthTracking
+  //                    [4, 6, ...] << lengthTrackingWithOffset
+
+  function div3(n) {
+    return Number(n) % 3 == 0;
+  }
+  function even(n) {
+    return Number(n) % 2 == 0;
+  }
```
Comment (ptomato) [inline test/built-ins/Array/prototype/some/resizable-buffer.js:36]: Not used?

File: test/built-ins/TypedArray/prototype/some/resizable-buffer.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  //                    [4, 6] << fixedLengthWithOffset
+  //              [0, 2, 4, 6, ...] << lengthTracking
+  //                    [4, 6, ...] << lengthTrackingWithOffset
+
+  function div3(n) {
+    return Number(n) % 3 == 0;
+  }
+  function even(n) {
+    return Number(n) % 2 == 0;
+  }
```
Comment (ptomato) [inline test/built-ins/TypedArray/prototype/some/resizable-buffer.js:36]: Not used?

**Review by ptomato — APPROVED** [review]
Minor nitpick.

File: test/built-ins/Array/prototype/some/resizable-buffer.js (hunk 1 above)
Comment (ioannad) [reply test/built-ins/Array/prototype/some/resizable-buffer.js]: Done

File: test/built-ins/TypedArray/prototype/some/resizable-buffer.js (hunk 2 above)
Comment (ioannad) [reply test/built-ins/TypedArray/prototype/some/resizable-buffer.js]: Removed.

**Pull request merged**